src/DistributedLockCodeGen/DistributedLockCodeGen.csproj

```diff
@@ -10,9 +10,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="nunit" Version="3.12.0" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.15.1" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
+    <PackageReference Include="nunit" />
+    <PackageReference Include="NUnit3TestAdapter" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" VersionOverride="16.2.0" />
```

Comment:
This should not be necessary but was done intentionally to minimize the diff in the lock file. (You'll see that the only diff is `version`. Do you agree that we should remove `VersionOverride` here?

Reply:
Good callout. I think this version should be the same as what we have in the Tests project, so no need for an override.